springwolf-add-ons/springwolf-kotlinx-serialization-model-converter/src/main/java/io/github/springwolf/addons/kotlinx_serialization_model_converter/configuration/KotlinxSerializationModelConverterAutoConfiguration.java

```diff
@@ -3,18 +3,20 @@
 
 import io.github.springwolf.addons.kotlinx_serialization_model_converter.converter.KotlinxSerializationModelConverter;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 /**
  * Spring AutoConfiguration adding an {@link KotlinxSerializationModelConverter} Bean to the spring context.
  */
 @Configuration(proxyBeanMethods = false)
+@ConditionalOnProperty(name = "springwolf.enabled", havingValue = "true", matchIfMissing = true)
 public class KotlinxSerializationModelConverterAutoConfiguration {
 
     @Bean
     public KotlinxSerializationModelConverter kotlinxSerializationTypeConverter(
-            @Value("${springwolf.use-fqn}") boolean useFqn) {
+            @Value("${springwolf.use-fqn:true}") boolean useFqn) {
```

Comment:
I was not aware that we have an implicit dependency to core configuration here. For me SpringwolfConfigProperties is the source of truth for configuration. Therefore I would suggest to add the dependency to core and use the the properties object.

Reply:
I thought about that, but I was hesitant to couple both modules. I will do the changes later today.